src/App.jsx

```diff
@@ -14,45 +16,87 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const SORT_ALPHABET = 'alphabet';
+const SORT_BY_LENGTH = 'length';
+
+const getSortedGoods = (goods, { sort, reverse }) => {
+  if (sort) {
+    goods.sort((a, b) => {
+      switch (sort) {
+        case SORT_ALPHABET:
+          return a.localeCompare(b);
+
+        case SORT_BY_LENGTH:
+          return a.length - b.length;
+
+        default:
+          return 0;
```

Comment:
It would be great to move switch...case out of the sort callback so you don't need to call it if the case is default